requirements/base.txt

```diff
@@ -11,7 +11,8 @@ pexpect==4.6.0
 psutil==5.3.1
 psycopg2-binary==2.7.4
 PTP==0.4.2
-pycurl==7.43.0.2
+pycurl==7.43.0
+pyhamcrest
```

Comment:
@NishealJ are these changes required?

Reply:
Hi @sharmamohit123 ,
while running the test at the local instance, we have to install the test.txt requirements and then run the tests(while running it locally it was suggesting to install pyhamcrest) we can update the dev docs.

I've removed this in my latest commits